apps/music/js/ui/views/list_view.js

```diff
@@ -348,11 +348,12 @@ var ListView = {
     var data = this.dataSource[index];
     var keyRange = (target.dataset.keyRange != 'all') ?
       IDBKeyRange.only(target.dataset.keyRange) : null;
-    var title = data.metadata.title;
+    var title = data.metadata.title ? data.metadata.title :
+        data.metadata.l10nId;
```

Comment:
Minus the code error:
````
var l10nId = data.metadata.l10nId;
````

It sounds like a better idea. Will fix that.

Thanks